Sources/Knit/ServiceCollection/Container+ServiceCollection.erb

```diff
@@ -1,158 +0,0 @@
-//
-// NOTICE:
-//
-// This file is generated from Container+ServiceCollection.erb by ERB.
-// Do NOT modify it directly.
-// Instead, modify Container+ServiceCollection.erb and run `Scripts/gencode`.
-//
-<% type_count = 20 %>
-
-import Foundation
-
-extension Container {
-
-    /// Registers a service factory into a collection.
-    ///
-    /// Usage:
-    /// ```
-    /// let container = Container()
-    /// container.addBehavior(ServiceCollector())
-    /// container.registerIntoCollection(Animal.self) { _ in Cat(...) }
-    /// container.registerIntoCollection(Animal.self) { _ in Dog(...) }
-    ///
-    /// let animals = resolver.resolveCollection(Animal.self)
-    /// print(animals.entries) // [Cat, Dog]
-    /// ```
-    /// - Parameters:
-    ///   - service: The service type to register.
-    ///   - factory: The closure to specify how the service type is resolved with the dependencies of the type.
-    ///              It is invoked when the ``Container`` needs to instantiate the instance.
-    ///              It takes a ``Resolver`` to inject dependencies to the instance,
-    ///              and returns the instance of the component type for the service.
-    /// - Returns: The registered service entry.
-    @discardableResult
-    public func registerIntoCollection<Service>(
-        _ service: Service.Type,
-        factory: @escaping @MainActor (Resolver) -> Service
-    ) -> ServiceEntry<Service> {
-        self.register(
-            service,
-            name: makeUniqueCollectionRegistrationName(),
-            factory: { resolver in
-                MainActor.assumeIsolated {
-                    return factory(resolver)
-                }
-            }
-        )
-    }
```

Comment:
We still want to keep this part of the erb

Reply:
There's no need for the erb anymore since it's only a single function left. It's just coded directly now.

Reply:
Ah sorry missed that part, merge away!